redash/models/__init__.py

```diff
@@ -772,6 +772,25 @@ def evaluate(self):
     def subscribers(self):
         return User.query.join(AlertSubscription).filter(AlertSubscription.alert == self)
 
+    def render_template(self):
+        data = json_loads(self.query_rel.latest_query_data.data)
+        context = {'rows': data['rows'], 'cols': data['columns']}
+        return mustache_render(self.options['template'], context)
+
+    @property
+    def template(self):
+        if 'template' in self.options:
+            return self.options['template']
+        else:
+            return ""
+
+    @property
+    def custom_subject(self):
+        if 'subject' in self.options:
+            return self.options['subject']
+        else:
+            return ""
```

Comment:
You can simply call `self.options.get('subject', '')`.

Reply:
thanks. I've fixed.